back-end/src/service/loginService.js

```diff
@@ -3,18 +3,17 @@ const { User } = require('../database/models');
 
 const loginServices = async (email, password) => {
   const findUser = await User.findOne({ where: { email } });
-  
+
   if (!findUser) {
-    return { code: 404, message: 'Not found' };
+    return { code: 404, message: 'Não encontrado' };
```

Comment:
Seria legal talvez especificar o que não foi encontrado, ou nesse caso se as validações dos campos já tiverem sido feitas pelos middlewares, poderíamos até retornar uma mensagem de ''usuário não existe'' ou ''usuário não cadastrado''

Reply:
Concordo, @jonathanjf!
Vejam se essa sugestão faz sentido:
```suggestion
    return { code: 404, message: 'Não existe nenhuma pessoa cadastrada com esse e-mail' };
```

Reply:
beleza Jonathan, faz total sentido mesmo